service/src/main/java/uk/gov/hmcts/reform/fpl/service/additionalapplications/UploadAdditionalApplicationsService.java

```diff
@@ -81,11 +82,21 @@ public AdditionalApplicationsBundle buildAdditionalApplicationsBundle(CaseData c
         final LocalDateTime currentDateTime = time.now();
 
         List<Element<Other>> selectedOthers = new ArrayList<>();
+        List<Element<Respondent>> selectedRespondents = new ArrayList<>();
         if (featureToggleService.isServeOrdersAndDocsToOthersEnabled()) {
-            selectedOthers = othersService.getSelectedOthers(caseData.getAllOthers(),
-                caseData.getOthersSelector(), caseData.getNotifyApplicationsToAllOthers());
+            selectedOthers = peopleInCaseService.getSelectedOthers(
```

Comment:
Would it make sense to pass the caseData as the only parameter into this method?

Reply:
yeah, updated to pass casedata